node-bb/README.md

```diff
@@ -0,0 +1,131 @@
+# TripleA Forums
+
+The software that powers the TripleA Forums is [NodeBB](https://github.com/NodeBB/NodeBB). 
+This document is instructions on how to build and run the Docker container of the forums using an 
+[official Docker container of NodeBB](https://hub.docker.com/r/nodebb/docker) as the base image.
+
+### Building
+
+1. Make a directory on your machine for where the config file and upload folder will live. ```mkdir -p /opt/triple-forums```
+1. Make a directory for uploaded files. Inside the upload directory, create folders category, emoji, files, profile, 
+sounds, and system.
+    ```
+    mkdir -p /opt/triple-forums/uploads/category
+    mkdir -p /opt/triple-forums/uploads/emoji
+    mkdir -p /opt/triple-forums/uploads/files
+    mkdir -p /opt/triple-forums/uploads/profile
+    mkdir -p /opt/triple-forums/uploads/sounds
+    mkdir -p /opt/triple-forums/uploads/system
+    ```
+1. Build the Docker image using the command ```docker build -t triplea-forums:latest .```
+
+### Running
+
+**First time run**
+
+You will need a running database. The database can be either Redis, Mongo, or Postgres. Make sure you have one of these
```

Comment:
Which DB did you use @bacrossland ? Any thoughts to check in a docker file or a docker-compose for the DB piece too?

Reply:
I just ran a docker container for Redis on my system. Nothing custom. I'll update the readme to put in instructions for running all three types of databases with a volume mount for keeping the data permanent.

Reply:
IMO you can stick to mongo, we run that in production. The docker should simulate prod as much as possible so we can tes it as best as possible locally before deploying.

Reply:
I updated the README.md and the config.json to be Mongo focussed only.